### PR TITLE
Code mostly Constant-ized

### PR DESCRIPTION
**Pull request by kobliha**
- New ConfigAgent::Constant (to easily port to another distro later)
- Some paths to config files also constant-ized (not only) for the same reason

### CODE REVIEW

**Comment by mvidner** [comment]
Nice, except one ugly part: `ConfigAgent::Constant::FOO_BAR` should be abbreviated to `ConfigAgent::FOO_BAR` since the uppercase already indicates to the reader that it is a constant.

File: libconfigagent/lib/config_agent/constant.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+#++
+
+module ConfigAgent
+  module Constant
+    AGENTS_LOGDIR = '/var/log/config_agents'
+    AGENTS_LOGFILE = 'services.log'
+
+    LENSES_DIR = '/usr/share/augeas/lenses/'
```
Comment (jreidinger) [inline libconfigagent/lib/config_agent/constant.rb]: this should go to augeas wrapper class. I think we should have constants near related code.

File: libconfigagent/lib/config_agent/constant.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+#
+#   You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+#++
+
+module ConfigAgent
+  module Constant
+    AGENTS_LOGDIR = '/var/log/config_agents'
+    AGENTS_LOGFILE = 'services.log'
```
Comment (jreidinger) [inline libconfigagent/lib/config_agent/constant.rb]: Both of them should go to logger. I think we should have constants near related code. So constants related to logs should be in log class. It is easier to read it then one big file full of constants.

File: libconfigagent/lib/config_agent/constant.rb (hunk 1 above)
Comment (kobliha) [inline libconfigagent/lib/config_agent/constant.rb]: This is actually a bit more difficult. AugeasWrapper class inherits from FileAgent and the LENSES_DIR is used in several classes (Adjtime, SshConfig and later maybe more) that also inherit from FileAgent. So, maybe I could put it to FileAgent class instead.

File: libconfigagent/lib/config_agent/constant.rb (hunk 2 above)
Comment (kobliha) [inline libconfigagent/lib/config_agent/constant.rb]: Good idea

**Comment by kobliha** [comment]
Done, ready for review / merging

File: libconfigagent/lib/config_agent/constant.rb (hunk 1 above)
Comment (jreidinger) [inline libconfigagent/lib/config_agent/constant.rb]: We plan to start using AugeasWrapper for all augeas based agents

**Pull request merged**